packages/mux-player/src/styles.css

```diff
@@ -8,6 +8,11 @@
   display: none;
 }
 
+mux-video::part(video) {
```

Comment:
question(non-blocking): should this go into `mux-video`?

Reply:
right maybe.

and I also just saw `object-fit: fill` is the default CSS value but media-poster-image was already set to cover so best to have those match right?

Reply:
scratch that `object-fit: contain;` is the default for video in browsers

![SCR-20221007-jmo](https://user-images.githubusercontent.com/360826/194636395-fa8c1e06-7ead-4ed5-b910-44fe3a500f59.png)